### ISSUE TITLE
OUT OF SCOPE: Component - Image Slider - Various issues

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/Adapt-accessibilityfixes#19]
We do not recommend re-enabling this component, but if we do:

• Component uses image path as alt text. 
• No accessible name for the next and previous buttons.
• Doesn’t read the text that change above the new images.

No labels on buttons, no indication that the image has changed, titles not read out when changed, image alt text is not read or difficult to reach the image to get the alt text
